Upgrade: don't show error during upgrade when analyzing external URL import

RobinMalfait opened the pull request:
This PR improves the output of the upgrade tool when we are handling imports and the import happens to be an external URL.

External URLs shouldn't and can't be upgraded, so printing an error message doesn't help the user.

Additionally, if an `@import` is using the `url(…)` function, then we skip over it and continue with the rest of the imports. 

| Before | After |
| --- | --- |
| <img width="1455" alt="image" src="https://github.com/user-attachments/assets/1ee00ea4-68e1-4252-b1cf-30a04f608b75"> | <img width="1455" alt="image" src="https://github.com/user-attachments/assets/da1f3eaf-dedb-4b1b-bf73-93bdfee65759"> |

Running this on github.com/parcel-bundler/parcel

| Before | After |
| -- | -- |
| <img width="1552" alt="image" src="https://github.com/user-attachments/assets/89987444-8008-4edd-a907-6ad9276a86a0"> | <img width="1552" alt="image" src="https://github.com/user-attachments/assets/cc2a34ae-ef17-4ad1-b06d-097874400b4d"> |

RobinMalfait committed:
do not show error when analyzing external URL imports

RobinMalfait committed:
update changelog

philipp-spiess commented on CHANGELOG.md:
```diff
@@ -20,6 +20,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Ensure `flex` is suggested ([#15014](https://github.com/tailwindlabs/tailwindcss/pull/15014))
 - _Upgrade (experimental)_: Resolve imports when specifying a CSS entry point on the command-line ([#15010](https://github.com/tailwindlabs/tailwindcss/pull/15010))
 - _Upgrade (experimental)_: Resolve nearest Tailwind config file when CSS file does not contain `@config` ([#15001](https://github.com/tailwindlabs/tailwindcss/pull/15001))
+- _Upgrade (experimental)_: Improve output when running into issues ([#15038](https://github.com/tailwindlabs/tailwindcss/pull/15038))
```
Should we be more specific here? 

```suggestion
- _Upgrade (experimental)_: Improve output when CSS imports can not be found ([#15038](https://github.com/tailwindlabs/tailwindcss/pull/15038))
```

RobinMalfait committed:
ignore errors, and skip handling `@import`

RobinMalfait committed:
remove unused code

This code wasn't used, so let's clean it up.

RobinMalfait committed:
add import with external URL

This guarantees that:
1. The `@import` is not modified in any way, shape or form
2. The `@import` doesn't crash the rest of the system
3. Imports later in the file are still migrated as expected
